src/Ombi/ClientApp/src/app/settings/emby/emby.component copy.html

```diff
@@ -0,0 +1,110 @@
+﻿
```

Comment:
Is this file needed?

Reply:
The copy is there as a failsafe. I made changes to the emby.component.html to make it use mat-tab-group. But I don't have Emby running so I am unable to check if it is working as intended.